stable/kong/templates/deployment.yaml

```diff
@@ -17,10 +17,13 @@ spec:
       component: app
   template:
     metadata:
-    {{- if .Values.podAnnotations }}
       annotations:
+        {{- if (and (not .Values.ingressController.enabled) (eq .Values.env.database "off" )) }}
+        checksum/dbless.config: {{ toYaml .Values.dblessConfig.config | sha256sum }}
```

Comment:
You need to take into account that `config` can be empty here.

Reply:
ok! changed